frontend/tests/unit/router.spec.ts

```diff
@@ -1,7 +1,38 @@
 import firebase from "firebase/app";
-import { routerBeforeEach } from "@/router";
+import { checkAuth, redirectSections } from "@/router";
+import { mockClaims } from "./components/HomePageButtons.spec";
 
-describe("routerBeforeEach", () => {
+describe("redirectSections", () => {
+  let to: any;
+  let from: any = {};
+  let next: any;
+
+  beforeEach(() => {
+    next = jest.fn();
+  });
+
+  test.each`
+    claims                   | expectedPath
+    ${{ TE: true }}          | ${"/te/my"}
+    ${{ coordinator: true }} | ${"/te/tasks"}
+  `(
+    "should redirect to first available child route that matches claims $claims",
+    async ({ claims, expectedPath }) => {
+      mockClaims(claims);
+      to = {
+        fullPath: "/te",
```

Comment:
Move this path to the test data table as well. This way you will be able to test various routes and claims combination.

Reply:
By the way, can't we get the whole `to` from the real router by path? Why duplicating here?